rocsp/rocsp.go

```diff
@@ -176,7 +180,13 @@ func (c *Client) GetResponse(ctx context.Context, serial string) ([]byte, error)
 
 	responseKey := MakeResponseKey(serial)
 
-	val, err := c.rdb.Get(ctx, responseKey).Result()
+	resp := c.rdb.Get(ctx, responseKey)
+	// go-redis `Get` returns redis.Nil error when key does not exist. In
+	// that case return a ErrNotFound error.
+	if errors.Is(resp.Err(), redis.Nil) {
```

Comment:
I think you would get the same result if you kept `val, err := c.rdb.Get(ctx, responseKey).Result()`, and made this:

    if errors.Is(err, redis.Nil)

Is that not the case? Then you could integrate this with the existing error-checking stanza, which seems cleaner.

Reply:
Yes, thanks. Done.